Mathlib/Analysis/LocallyConvex/AbsConvex.lean

```diff
@@ -72,6 +163,61 @@ theorem nhds_basis_abs_convex_open :
 
 end NontriviallyNormedField
 
+section
+
+variable (𝕜) [NontriviallyNormedField 𝕜]
+variable [AddCommGroup E] [Module ℝ E] [Module 𝕜 E]
+
+theorem AbsConvex.hullAdd {s t : Set E} :
+    absConvexHull 𝕜 (s + t) ⊆ absConvexHull 𝕜 s + absConvexHull 𝕜 t :=
+  absConvexHull_min (add_subset_add (subset_absConvexHull 𝕜 s) (subset_absConvexHull 𝕜 t))
+    ⟨Balanced.add (balanced_absConvexHull 𝕜 s) (balanced_absConvexHull 𝕜 t),
+      Convex.add (convex_absConvexHull 𝕜 s) (convex_absConvexHull 𝕜 t)⟩
+
+theorem absConvexHull_eq_convexHull_balancedHull [SMulCommClass ℝ 𝕜 E] {s : Set E} :
+    absConvexHull 𝕜 s = convexHull ℝ (balancedHull 𝕜 s) := le_antisymm
```

Comment:
`absConvexHull 𝕜 s = balancedHull ℝ (convexHull ℝ s)` is also true, right? Can you prove it?

Reply:
Wikipedia claims to have a counter example: https://en.wikipedia.org/wiki/Absolutely_convex_set#Examples

Reply:
Ah yeah of course. Can you leave a comment explaining that?

Reply:
I've shown `balancedHull 𝕜 (convexHull ℝ s) ⊆ absConvexHull 𝕜 s` and added a note about equality not holding in general.